solr/core/src/java/org/apache/solr/util/StartupLoggingUtils.java

```diff
@@ -187,4 +187,19 @@ public static String getLogLevelString() {
     else if (rootLogger.isErrorEnabled()) return "ERROR";
     else return "INFO";
   }
+
+  /**
+   * Check whether Jetty requestlogging is enabled and log info about it
+   */
+  public static void checkRequestLogging() {
+    boolean requestLogEnabled = System.getProperty("sun.java.command", "").contains("module=requestlog");
```

Comment:
Found no smarter way to check whether that module is active. There is probably some static Jetty class method that would do the same and be more rubust (such as when Oracle changes the name of `sun.java.command` property..?

Reply:
If we are executing somewhere that has access to the `Server` object (like the tests `JettySolrRunner`) then we can check if `server.getRequestLog()` returns null. I haven't figured out how to get access to our server object outside of that though. This is what I get when looking at who has a reference to the Server object and none of those seem particularly approachable.

<img width="580" alt="image" src="https://user-images.githubusercontent.com/1592330/150230472-4e79680f-255c-4fa8-ac24-89d46a87cc33.png">

Reply:
I suspect that being able to arbitrarily load the server object would be some kind of security constraint violation, but I haven't really dug into it much further.

Reply:
Could of course edit bin/solr and bin/solr.cmd to add a custom system property for this (should only be a few extra lines), but I don't think it's worth it. Worst case is that this logging disappears if the string pattern in command line args changes.

Reply:
Ok, I found a trick, where I could set a sysprop in `solr/server/etc/jetty-requestlog.xml` by some xml tags, so now I check on that instead.